tests/lib/network_helper.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Jan 23 10:45:21 2019
+
+@author: gerben
+"""
+
+from netaddr import *
+import socket
+import struct
+import re
+class mac_custom(mac_unix):
+    pass
+
+def mac_format(mac_addr):
+    mac_tmp = re.sub("[\s+\.\!\-\:\/_,$%^*(+\"\')]+|[+()?【】“”！，。？、~@#￥%……&*（）]+',","",mac_addr)
```

Comment:
We need to be explicit about what the input and output format is, if not we can add a try / except or for loop to try to find the right one... but this is too generic.

Ideally we just use word_sep in the class above for the cases we want to detect.